gdk/wizard/WizardChecker.py

```diff
@@ -77,3 +83,70 @@ def is_valid_custom_build_command(self, input_value):
             return False
         except ValueError:
             return len(input_value) > 0
+
+    def check_build_options(self, input_value):
+        try:
+            # Convert the input string to a dictionary (object)
+            input_obj = json.loads(input_value)
+        except json.JSONDecodeError:
+            return False
+
+        if not isinstance(input_obj, dict):
+            return False
+
+        if len(input_obj) > 2:
```

Comment:
It is ok if customers want to specify an empty object for build options. ie. {} is still a valid input IMO.